std/traits.d

```diff
@@ -797,12 +797,13 @@ private template fqnType(T,
     {
         alias PSC = ParameterStorageClass;
 
-        return format("%s%s%s%s%s",
+        return format("%s%s%s%s%s%s",
```

Comment:
Format is an overkill for pure concatenation and has pretty bad CTFE performance.
(I'm aware it was like this)

Reply:
This whole template is a bit useless... It could (should) just call the trait directly, since this already returns a tuple of strings...

Reply:
I'd say just change this to `std.conv.text` or pure string concatenation and let's get this PR merged.

Reply:
Done